garfield/lookup/models.py

```diff
@@ -0,0 +1,3 @@
+from django.db import models
+
+# Create your models here.
```

Comment:
Should we save lookups and their result in a model here?

Reply:
It seems redundant to do this as we are basically storing the same information we are looking up. But if you want use to we can